test/fixtures/rippled/accountTx.js

```diff
@@ -1,30 +1,34 @@
 /* eslint-disable max-len */
-'use strict';
```

Comment:
Looks like a bunch of fixtures got caught up in this too - can you undo the changes for the JS files?